js/apps/system/_admin/aardvark/APP/frontend/js/views/modalView.js

```diff
@@ -294,7 +295,12 @@
         }
       }
       _.each(buttons, function (b, i) {
-        if (b.disabled || !b.callback) {
+        if (b.disabled || b.type === 'disabled' || !b.callback) {
+          if (divID) {
+            $('#modalButton' + i).attr('disabled', true);
+          } else {
+            $('#' + divID + ' ' + '#modalButton' + i).attr('disabled', true);
```

Comment:
Why is `divID` uses here, in case it is falsy?
This does not seem right. Are the `if` and `else` branches confused?